### ISSUE TITLE
Add ACF fields to video Module

### ISSUE THREAD

**Issue by aaronmeeusPK** [beaverbuilder/feature-requests#367]
**Is your feature request related to a problem? Please describe.**
There is no option to use an acf field for the video module. this way we have to build it custom.

**Describe the solution you'd like**
The possibility to connect an acf field to the module. 

**Additional context**
Please! :)

**Comment by codente** [comment]
We are looking into adding this - there's a little bit to figure out first but I think it should be doable.

**Comment by zackpyle** [comment]
@codcode What about the poster?

**Comment by aaronmeeusPK** [comment]
> You can if you use the embed option. However, there's a little bit to figure out to enable it for the other fields for mp4 and webm but I think it should be doable.

Yes that's true. But this was specific for the not embed option ^^

**Comment by zackpyle** [comment]
Yeah, if they can get it working with mp4/webm, that'd be cool. 

Adding Themer support for the poster will be an easy addition though.

**Comment by codente** [comment]
> > You can if you use the embed option. However, there's a little bit to figure out to enable it for the other fields for mp4 and webm but I think it should be doable.
> 
> Yes that's true. But this was specific for the not embed option ^^

Your request doesn't specify which one it's for actually so that's why I mentioned it.

**Comment by aaronmeeusPK** [comment]
I am really sorry! :)

**Comment by codente** [comment]
This was added in BB 2.8.4
